### ISSUE TITLE
Typo in Intro to Deep Learning exercise

### ISSUE THREAD

**Issue by Rashi181** [Kaggle/learntools#484]
There is a small typo in the exercise for stochastic gradient descent. 
"To get a better understanding of these two parameters, we'll look at the linear model, our ppsimplest neural network."
It should be "simplest" not "ppsimplest".

<img width="933" height="791" alt="Image" src="https://github.com/user-attachments/assets/0dd8b472-6870-48b1-b2db-a6dc6cd389e6" />

**Comment by Aditya-Sajgotra** [comment]
can i take this?

**Comment by Rashi181** [comment]
> can i take this?

Sure
